### ISSUE TITLE
verb to disable or reduce nutrition gain on self clean function for janitor borgs.✨

### ISSUE THREAD

**Issue by crabcube** [CHOMPStation2/CHOMPStation2#8709]
#### Brief description of the feature
Now that janitor cyborgs can use the self clean to get glass to refill their light replacer. I have discovered that using the self clean feature on trash leaves you constantly slowed by nutrition. But I also know the slow is to deal with borg powergaming. but as someone who is literally just trying to clean, this blows.

#### What you want to happen
 Janitor cyborgs can opt out of nutrition so they're not crippled when getting material from garbage. EXCLUSIVELY for /obj/item/trash, to prevent the destruction of all maint items. and EXCLUSIVELY in the trash compactor belly, not vore bellies. 
 
 In summary, /obj/item/trash in the Garbage processor borg belly, not supplying nutrition at all.

#### Anything else you may wish to add:
- (Suggestion just for janitor cyborgs because self clean is essential to their updated kit.)

**Comment by Kashargul** [comment]
You can use glass sheets as intended.
